lib/OctokitAction.ts

```diff
@@ -30,6 +34,13 @@ export abstract class OctokitAction extends Action {
     return this.graphqlWithAuth(query);
   }
 
+  protected async moveIssue(issueId: string, transitionName: string): Promise<void> {
```

Comment:
Please move this after `getPullRequest`, before `sendSlackMessage`. To keep some logical grouping of the methods